.github/workflows/codeql-daily.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@2b46439dd5477d8a1659811cdc500d35e601a1cb # v1.0.32
+      uses: github/codeql-action/init@474bbf07f9247ffe1856c6a0f94aeeb10e7afee6 # v1.0.32
```

Comment:
@phlax what do you think about keeping the comments up-to-date?

Reply:
i think probs just remove them